tests/core/fsck-on-boot/task.yaml

```diff
@@ -0,0 +1,86 @@
+summary: the boot base provides essential fsck programs
+# TODO:UC20: this test fails on core16, core18 and core20 and may require
```

Comment:
lol the reason this doesn't work on uc20 is by design I had misunderstood the requirements of fsck for vfats, see

https://github.com/snapcore/snapd/blob/c8573086fe4ad024bf2b573527f64ec1a657487a/cmd/snap-bootstrap/cmd_initramfs_mounts.go#L478-L479

I will file a patch after this lands to change that and adjust this test accordingly, thanks for the test

Reply:
I pushed the patch here, this should be good